mkosi/resources/man/mkosi.1.md

```diff
@@ -1016,10 +1016,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 
 `KernelModulesInclude=`, `--kernel-modules-include=`
 :   Takes a list of regex patterns that specify kernel modules to include in the image. Patterns should be
-    relative to the `/usr/lib/modules/<kver>/kernel` directory. mkosi checks for a match anywhere in the module
-    path (e.g. `i915` will match against `drivers/gpu/drm/i915.ko`). All modules that match any of the
-    specified patterns are included in the image. All module and firmware dependencies of the matched modules
-    are included in the image as well.
+    relative to the `/usr/lib/modules/<kver>/kernel` directory (note that they match against modules in the
+    `updates` and `extra` sibling directories too). mkosi checks for a match anywhere in the module path (e.g.
```

Comment:
I think this would be clearer:
```suggestion
    relative to the `/usr/lib/modules/<kver>` directory. mkosi checks for a match anywhere in the module path (e.g.
```
mentioning the `updates` and `extra` directories explicitly only obscures that, we look relative to that.